domene/src/main/kotlin/no/nav/tiltakspenger/vedtak/Innsending.kt

```diff
@@ -305,6 +319,19 @@ class Innsending private constructor(
         tilstand.håndter(this, foreldrepengerMottattHendelse)
     }
 
+    fun håndter(overgangsstønadMottattHendelse: OvergangsstønadMottattHendelse) {
+        if (journalpostId != overgangsstønadMottattHendelse.journalpostId()) return
+        // Den påfølgende linja er viktig, fordi den blant annet kobler hendelsen sin aktivitetslogg
+        // til Søker sin aktivitetslogg (Søker sin blir forelder)
+        // Det gjør at alt som sendes inn i hendelsen sin aktivitetslogg ender opp i Søker sin også.
+        kontekst(overgangsstønadMottattHendelse, "Registrert OvergangsstønadMottattHendelse")
+        if (erAlleFaktaInnhentet()) {
```

Comment:
Nå ble jeg litt usikker, jeg tror jeg må åpne koden i IntelliJ for å se på flyten. Men er det riktig å stoppe opp her hvis alle fakta er innhentet? Vil du da håndtere oppdateringer?

Reply:
Nå har jeg pulla branchen, og jeg ville droppe den erAlleFaktaInnhentet sjekken her. Jeg ser at den er der for fun håndter(foreldrepengerMottattHendelse: ForeldrepengerMottattHendelse) og fun håndter(uføreMottattHendelse: UføreMottattHendelse) også, men jeg ville skrevet det vekk derfra også.